src/components/Card/Card.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from "react";
+import React from 'react'
+import { CardStyled, CardHeader, CardFooter } from "./Card.styles";
+import { CardProps } from "./types";
+import {iconTypes} from "../Icon/collection";
+import colors from "../../styles/colors";
+import {Icon} from "../Icon";
+import {Tooltip} from "../Tooltip";
+
+const Card: React.FC<CardProps> = ({
+    id,
+    children,
+    description,
+    selected,
+    title,
+    tooltipText = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores corporis vero in ipsum temporibus nesciunt!",
+}: CardProps) => {
+    const [ isSelected, setSelected ] = useState<boolean | undefined>(selected);
+    const [showCheckedIcon, toggleChecked] = useState<boolean | undefined>(false);
+
+    useEffect(() => {
+        toggleChecked(selected);
+    }, [selected]);
+
+    return (
+        <CardStyled
+            id={id}
+            data-testid={"card-test-id"}
+            onClick={() => {
+                setSelected(!isSelected);
+                toggleChecked(!showCheckedIcon);
+            }}
+            selected={isSelected}
+        >
+            <CardHeader data-testid={"header-test-id"}>
+                { showCheckedIcon &&
+                    <Icon
+                    data-testid={"check-test-id"}
+                    fill={colors.green}
+                    size={24}
+                    svg={iconTypes.checkmark}
+                    />
+                }
+                <Tooltip
```

Comment:
maybe tooltip should not render if there is no tooltipText prop passed, rather than rendering Lipsum. Or make it not optional if you need one

Reply:
made it not optional